torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1536,6 +1565,36 @@ PyTorchModelLoader::createGlowConstantFromIValue(
   return F_.getParent()->createConstant("constant", std::move(t));
 }
 
+llvm::Error
+PyTorchModelLoader::createPlaceholderFromTensor(const torch::jit::Value *value,
+                                                glow::Placeholder *&PH,
+                                                const at::Tensor &ptTensor) {
```

Comment:
don't need tensor, just neet the tensor type. Why not just create the tensor type then call loadValue() like before? This is just adding a new method to do that

Reply:
because in each case we start with tensor and will call tensor -> tensor shape every time.